Removed line break

philpursglove opened the pull request:
Removed an extra line break as I think that's what's messing up the output of the page on the live site - see the last `deleteOperation` section at https://docs.microsoft.com/en-us/azure/cosmos-db/create-table-dotnet#review-the-code

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

philpursglove committed:
Removed line break

Removed an extra line break as I think that's what's messing up the output of the page on the live site - see the last `deleteOperation` section at https://docs.microsoft.com/en-us/azure/cosmos-db/create-table-dotnet#review-the-code
